fix: Hidden files exist in the search results

Kakueeen opened the pull request:
Fixed hidden file detection problem

Log: fix bug

Bug: https://pms.uniontech.com/bug-view-147401.html

Commit:
fix: QT_INSTALL_LIBS cant assign value by qmake flags

Log: QT_INSTALL_LIBS 在赋值前应该判断是否已经传入值

Kakueeen committed:
fix: Hidden files exist in the search results

Fixed hidden file detection problem

Log: fix bug

itsXuSt reviewed (dismissed):
[是否检查兼容性] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

itsXuSt committed:
chore: add installation of dconfig files.

if dconfig is not supported by dtkcore, install the config files
directly into the default path.

Log: fix compilation error when DTKCORE's version is less than 5.5.18

Johnson-zs committed:
fix: emblem performance and menu param problems for dfm-extension (#475)

1. Fix the missing urlist parameter in the context menu
2. Optimize emblem to extend CPU performance

Log: Optimize dfm-extension

Task: https://pms.uniontech.com/task-view-164863.html
